File: Go/main/215.go
package main

import  "sort"

func findKthLargest(nums []int, k int) int {
	sort.Sort(sort.Reverse(sort.IntSlice(nums)))
	for i,v := range nums{
		if(i+1==k) {
			return v
		}
	}
	return 0
}
func main()  {
	nums := []int{3,2,3,1,2,4,5,5,6}
	k :=4
	findKthLargest(nums,k)
}

File: JS/215.js
var findKthLargest = function(nums, k) {
    r = 0
    nums.sort((a,b)=>b-a)
    nums.forEach((el,i)=>{
        if(i+1==k)
        {
            r = el
        }
    })
    return r
};

findKthLargest([3,2,3,1,2,4,5,5,6],4)